krun/platform.py

```diff
@@ -132,26 +133,63 @@ def _collect_dmesg_lines(self):
     def _timestamp_to_str(self, lt):
         return time.strftime(ABS_TIME_FORMAT, lt)
 
+    def get_allowed_dmesg_patterns(self):
+        return []
+
+    def filter_new_dmesg_line(self, line, patterns):
+        for p in patterns:
+            if p.match(line):
+                debug("Allowed dmesg change: '%s'" % line)
+                return True  # allowed change of dmesg
+        return False
+
     def check_dmesg_for_changes(self):
         new_dmesg_time = localtime()
         new_dmesg = self._collect_dmesg_lines()
+        patterns = self.get_allowed_dmesg_patterns()
 
-        old_fn = self._timestamp_to_str(self.last_dmesg_time)
-        new_fn = self._timestamp_to_str(new_dmesg_time)
-        lines = [x for x in difflib.unified_diff(
-            self.last_dmesg, new_dmesg, old_fn, new_fn, lineterm="")]
+        rv = self._check_dmesg_for_changes(patterns,
+                                           self.last_dmesg,
+                                           self.last_dmesg_time, new_dmesg,
+                                           new_dmesg_time)
```

Comment:
So as to not have a long line. The standard way is to line up the next line with the open parenthesis.